### PR TITLE
Adding Bigtable Table.read_rows().

### PR DESCRIPTION
**Pull request by dhermes**
FYI @tseaver This is the last of the non-HappyBase PRs!

### CODE REVIEW

File: gcloud/bigtable/table.py [hunk 1]
```diff
@@ -255,6 +256,61 @@ def read_row(self, row_key, filter_=None):
...
+        :param start_key: (Optional) The beginning of a range of row keys to
+                          read from. The range will include ``start_key``. If
+                          left empty, will be interpreted as the empty string.
+
+        :type end_key: bytes
+        :param end_key: (Optional) The end of a range of row keys to read from.
+                        The range will not include ``end_key``. If left empty,
+                        will be interpreted as an infinite string.
+
+        :type filter_: :class:`.row.RowFilter`, :class:`.row.RowFilterChain`,
```
Comment (tseaver) [inline gcloud/bigtable/table.py]: Out of order with the declared signature.

**Comment by tseaver** [comment]
Except for the ordering issue I noted, LGTM.

File: gcloud/bigtable/table.py (hunk 1 above)
Comment (dhermes) [inline gcloud/bigtable/table.py]: Nice catch! Thanks.

**Comment by dhermes** [comment]
Fixed the ordering (and dropped the un-needed references to `RowFilterChain`, etc. since those are all subclasses of `RowFilter`)

**Pull request merged**